src/main.js

```diff
@@ -598,7 +624,10 @@ async function createWindow() {
     });
     mainWindow.loadFile(path.join(appData.path, "index.html"));
 
+    
     // load the translations
+    // make sure language is supported, otherwise load translations from fallback language
+    if(!i18next.options.supportedLngs.includes(userData.data.language)) await userData.set("language", i18next.options.fallbackLng[0])
```

Comment:
this is basically the fallback I would propose. If the users language is not part of the supported languages, the fallback language would be persisted in users settings and the app would just continue loading.